examples/serverless-with-firebase/.gitignore

```diff
@@ -0,0 +1,276 @@
+dist/
```

Comment:
Is this gitignore needed?

Reply:
I usually use gitignore template generator to get a gitignore file, that's why it's huge, but save some time running into gitignore related issue. Maybe I should make a clean gitignore file here?

Reply:
I use a different generator, see the result here: https://github.com/UnlyEd/serverless-plugin-dynamodb-backups/blob/master/.gitignore

I believe it's similar, but cleaner.

Reply:
@Vadorequest It's the same generator. The different is I added more templates for different IDE and environment.

Reply:
@timneutkens I would like to keep this `.gitignore` as it is. Not because the configurations are must-have, but to keep a generally working `.gitignore` would benefit anyone takes this boilerplate. And it's fairly easy to remove pieces people don't like. This ignore file makes no assumption that people have global ignore file setup for platform.
As this folder is a subfolder within this repo, is something is missing in this ignore file, it would only haunt the person when he copy this boilerplate.